packages/web-components/src/combobox/combobox.styles.ts

```diff
@@ -43,4 +44,12 @@ export const comboboxStyles: (context: ElementDefinitionContext, definition: Com
     .selected-value:active {
         outline: none;
     }
-`.withBehaviors(appearanceBehavior('filled', selectFilledStyles(context, definition)));
+`.withBehaviors(
+  appearanceBehavior('filled', selectFilledStyles(context, definition)),
+  forcedColorsStylesheetBehavior(
+    css`
+      .start, .end, .indicator, ::slotted(svg) {
```

Comment:
Looks like this is already set in the main select styles? Do we need to do this here still?

Reply:
updated